components/brave_extension/extension/brave_extension/webstore.ts

```diff
@@ -0,0 +1,18 @@
+const config = { attributes: true, childList: true, subtree: true }
+
+const callback = (mutationsList: MutationRecord[], observer: MutationObserver) => {
+  const button: Element = document.getElementsByClassName('g-c-R webstore-test-button-label')[0]
```

Comment:
the `g-c-R` portion of the selector here might be easily broken (if they modify styles; looks like they're using CSS in JS libs)... You may want to try matching on `div.webstore-test-button-label`